src/extension/src/controller.ts

```diff
@@ -167,6 +171,31 @@ export class Controller {
     }
   }
 
+  private static getProjectName() {
+    const userOutputPath = vscode.workspace
+      .getConfiguration()
+      .get<string>("wts.changeSaveToLocation");
+    const outputPath: string = userOutputPath ? userOutputPath : os.homedir();
+    const defaultAppName = "myApp";
+    let newAppName = "myApp";
```

Comment:
why not say newAppName = defaultAppName?

Reply:
yup, makes sense